committeeoversightapp/models.py

```diff
@@ -1 +1,18 @@
 from django.db import models
+from opencivicdata.legislative.models import Event, EventParticipant, EventDocument
+
+class HearingCategoryType(models.Model):
+    id = models.CharField(max_length=100, primary_key=True)
+    name = models.CharField(max_length=100, primary_key=False)
+
+    def __str__(self):
+        return u'({0}) {1}'.format(self.id, self.name)
+
+class HearingCategory(models.Model):
+    event = models.ForeignKey(Event, null=True, on_delete=models.CASCADE)
```

Comment:
ah, `on_delete` is now a required argument for Django 2.....

Reply:
the only world I've ever known...